app/graphql/types/business/applications.gql

```diff
@@ -0,0 +1,30 @@
+type BusinessApplication {
+    applicationId: ID
```

Comment:
This data does not look like what is being asked for, please check the spreadsheet

Reply:
I created this based on what @dtaylor7 given on slack channel, I will ignore that endpoint and match the fields with spreadsheet now